apps/antalmanac/src/components/RightPane/CoursePane/SchoolDeptCard.tsx

```diff
@@ -54,6 +55,10 @@ class SchoolDeptCard extends PureComponent<SchoolDeptCardProps> {
                     <Typography
                         noWrap
                         variant={this.props.type === 'school' ? 'h6' : 'subtitle1'}
+                        onClick={() =>
+                            this.setState({
+                                commentsOpen: !this.state.commentsOpen,
+                            })}
                         className={this.props.classes.text}
                     >
```

Comment:
Try not to do things like this; typographic elements like headers, paragraphs, etc. aren't meant to be clicked on or produce side effects as a result -- general WAI/ARIA stuff. The correct fix is to use the [`Accordion`](https://mui.com/material-ui/react-accordion/) MUI component. We shouldn't even have to manage the open/closed state manually.

For reference, here's my implementation of it: https://github.com/icssc/AntAlmanac/blob/aponia-deploy/src/components/Schedule/Department/Department.tsx